doc/common/openstack-release.md

```diff
@@ -1,32 +1,32 @@
-# CNTT OpenStack Baseline Release Selection
+# Anuket OpenStack Baseline Release Selection
 
-This section specifies policies for the selection of the next CNTT OpenStack baseline release and the number of releases that CNTT shall support:
+This section specifies policies for the selection of the next Anuket OpenStack baseline release and the number of releases that the Anuket specifications shall support:
 -	criteria for the triggering of the next baseline selection
--	criteria to use in choosing the next OpenStack release, and 
--	the number of OpenStack releases to be supported by CNTT
+-	criteria to use in choosing the next OpenStack release, and
+-	the number of OpenStack releases to be supported by Anuket specifications
 
-The selection of a new OpenStack baseline release will be associated with a new CNTT release and a whole set of documents (RA1, RI1 and RC1) with new versions.. Please note that while a new OpenStack release selection may only trigger updates to certain sections, all document releases will be complete and can be utilised on their own independent of previous releases.
+The selection of a new OpenStack baseline release will be associated with a new Anuket release and a whole set of documents (RA1, RI1 and RC1) with new versions. Please note that while a new OpenStack release selection may only trigger updates to certain sections, all document releases will be complete and can be utilised on their own independent of previous releases.
 
 ## Triggering Events for next release selection
 This section specifies events that may trigger the selection of the next OpenStack release.
 -	Complete change in architecture: OpenStack, OpenStack Service or major API change of an OpenStack RA-1 required service
--	New OpenStack features, services or projects needed by CNTT workloads
+-	New OpenStack features, services or projects needed by Anuket-compliant workloads
 -	Major Security Fix (not fixed through a patch; OpenStack or OS) that affect APIs
--	Current CNTT [OpenStack release]( https://releases.openstack.org/) entered “[Extended Maintenance]( https://docs.openstack.org/project-team-guide/stable-branches.html#maintenance-phases)” phase approximately 18 months ago
+-	Current Anuket [OpenStack release]( https://releases.openstack.org/) entered “[Extended Maintenance]( https://docs.openstack.org/project-team-guide/stable-branches.html#maintenance-phases)” phase approximately 18 months ago
 
 ## OpenStack Release Selection Committee
```

Comment:
Suggest that we delete this section as the Anuket Charter doesn't have any such requirements.

Reply:
Could you handle that in another PR?

Reply:
@pgoyal01 - yes, let us delete this, as absolute now. @EditKoselak, I agree, please create another PR to remove this paragraph.